mempool/mempool.go

```diff
@@ -32,10 +32,6 @@ const (
 	// inclusion when generating block templates.
 	DefaultBlockPrioritySize = 20000
 
-	// MinHighPriority is the minimum priority value that allows a
-	// transaction to be considered high priority.
-	MinHighPriority = dcrutil.AtomsPerCoin * 144.0 / 250
```

Comment:
This is a breaking change to a v1 module.  It should be kept, and optionally deprecated:

`// Deprecated: Use mining.MinHighPriority`